SwissTransferFeatures/UploadProgressView/TransferSessionManager.swift

```diff
@@ -51,6 +51,8 @@ class TransferSessionManager: ObservableObject {
     @Published var completedBytes: Int64 = 0
     @Published var totalBytes: Int64 = 0
 
+    var currentUploadUUID: String?
```

Comment:
```suggestion
   private(set) var currentUploadUUID: String?
```

Reply:
Nice catch! And I'll make the structs `@frozen`